webpack/assets/stylesheets/application/variables/_colors.scss

```diff
@@ -1,7 +1,6 @@
-// Colors
-$primary_color: #005c7e;
-$border_color: #0187b6;
-$brand_color: #005c7e;
```

Comment:
I assume this will break theme plugin too?

Reply:
It shouldn't break the theme, we didn't have any good solution for `scss` anyway.